spec.html

```diff
@@ -1095,30 +1095,540 @@ <h1>Well-Known Symbols</h1>
+        <p>Some ECMAScript operators deal only with integers in specific ranges such as <emu-eqn>-2<sup>31</sup></emu-eqn> through <emu-eqn>2<sup>31</sup> - 1</emu-eqn>, inclusive, or in the range 0 through <emu-eqn>2<sup>16</sup> - 1</emu-eqn>, inclusive. These operators accept any value of the Number type but first convert each such value to an integer value in the expected range. See the descriptions of the numeric conversion operations in <emu-xref href="#sec-type-conversion"></emu-xref>.</p>
+
+        <p>The Number::unit value is *1*.</p>
+
+        <emu-clause id="sec-numeric-types-number-unaryMinus">
+          <h1>Number::unaryMinus (_x_)</h1>
+          
+          <emu-alg>
+            1. If _x_ is *NaN*, return *NaN*.
+            1. Return the result of negating _x_; that is, compute a Number with the same magnitude but opposite sign.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-numeric-types-number-bitwiseNOT">
+          <h1>Number::bitwiseNOT (_x_)</h1>
+
+          <emu-alg>
+            1. Let _oldValue_ be ? ToInt32(? _x_).
+            1. Return the result of applying bitwise complement to _oldValue_. The result is a signed 32-bit integer.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-numeric-types-number-exponentiate">
+          <h1>Number::exponentiate (_base_, _exponent_)</h1>
+
+          <emu-alg>
+            1. Return the result of <emu-xref href="#sec-applying-the-exp-operator" title>Applying the ** operator</emu-xref> with _base_ and _exponent_ as specified in <emu-xref href="#sec-applying-the-exp-operator"></emu-xref>.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-numeric-types-number-multiply">
+          <h1>Number::multiply (_x_, _y_)</h1>
+
+          <emu-alg>
+            1. Return the result of <emu-xref href="#sec-applying-the-mul-operator" title>Applying the * operator</emu-xref> to _x_ and _y_ as specified in <emu-xref href="#sec-applying-the-mul-operator"></emu-xref>.
```

Comment:
These "Applying the ... operator" sections should move here and become the body of these Number:: algorithms.

Reply:
Done.